src/xdocs/config_annotation.xml

```diff
@@ -94,45 +94,53 @@ public String getNameIfPresent() { ... }
               <td>tokens to check</td>
               <td>subset of tokens
                 <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CLASS_DEF">
-                    CLASS_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
-                    INTERFACE_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#PACKAGE_DEF">
-                    PACKAGE_DEF</a>,
-                <a href=
-                    "apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ENUM_CONSTANT_DEF">
-                    ENUM_CONSTANT_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ENUM_DEF">
-                    ENUM_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF">
-                    METHOD_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CTOR_DEF">
-                    CTOR_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#VARIABLE_DEF">
-                    VARIABLE_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ANNOTATION_DEF">
-                    ANNOTATION_DEF</a>,
-                <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ANNOTATION_FIELD_DEF">
-                    ANNOTATION_FIELD_DEF</a>.
+                    CLASS_DEF</a>
+                , <a href=
+                    "apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
```

Comment:
Why did we split `href=` from the URL?

Reply:
Otherwise it would have crossed the lin length limit. So instead of breaking the string, I moved the whole string to a new line

Reply:
Did the CI or tests complain that this must be changed? We have overrides in checkstyle to allow links to be longer than 100 characters.

Reply:
Yes, checkstyle failed.

Reply:
@gaurabdg , please recheck this one more time. Line length violations are ignored  on lines where "http|href" are present. 
You just move coma from most right to most left, it does not make big difference for LineLength.

Please restore all other lines.

Reply:
It is indeed giving error
`                , <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
`

Reply:
The `http|href` exemption is for `non_main_files_check`

Reply:
hmm, yes it gives violation:
```
[WARNING]      [echo] Checkstyle started (checkstyle_checks.xml): 29/05/2020 08:17:18 AM
[INFO] [checkstyle] Running Checkstyle  on 1150 files
[ERROR] [checkstyle] [ERROR] /home/rivanov/java/github/romani/checkstyle/src/xdocs/config_annotation.xml:98: Line should not be longer than 100 symbols [RegexpSingleline]
[WARNING]      [echo] Checkstyle finished (checkstyle_checks.xml) : 29/05/2020 08:17:51 AM
[WARNING]      [echo] Checkstyle started (checkstyle_non_main_files_checks.xml): 29/05/2020 08:17:18 AM
[INFO] [checkstyle] Running Checkstyle  on 2128 files
[WARNING]      [echo] Checkstyle finished (checkstyle_non_main_files_checks.xml): 29/05/2020 08:17:51 AM
[WARNING]      [echo] Checkstyle started (checkstyle_resources_checks.xml): 29/05/2020 08:17:18 AM
[INFO] [checkstyle] Running Checkstyle  on 1835 files
[WARNING]      [echo] Checkstyle finished (checkstyle_resources_checks.xml): 29/05/2020 08:17:51 AM
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

✘-1 ~/java/github/romani/checkstyle [gaurabdg/cleanup-xdocs L|✚ 1] 
08:17 $ git diff
diff --git a/src/xdocs/config_annotation.xml b/src/xdocs/config_annotation.xml
index e879bd1..f656486 100644
--- a/src/xdocs/config_annotation.xml
+++ b/src/xdocs/config_annotation.xml
@@ -95,8 +95,7 @@ public String getNameIfPresent() { ... }
               <td>subset of tokens
                 <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CLASS_DEF">
                     CLASS_DEF</a>
-                , <a href=
-                    "apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
+                , <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
                     INTERFACE_DEF</a>
                 , <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#PACKAGE_DEF">
                     PACKAGE_DEF</a>

```

Reply:
but it is strange ... line with ANNOTATION_FIELD_DEF is far more than 100, and there is no violation on it.

Reply:
> line with ANNOTATION_FIELD_DEF before fix (of the same block ) is far more than 100, and there is no violation on it.

The `format` regex allows to since there's no comma at the beginning. I'll fix the regex to not throw any error for the new comma placement.